pom.xml

```diff
@@ -344,6 +354,9 @@
           <pomIncludes>
             <pomInclude>*/pom.xml</pomInclude>
           </pomIncludes>
+          <setupIncludes>
+            <setupInclude>setup_test-dependencies-transitive-optional-dep/pom.xml</setupInclude>
```

Comment:
Does this really need to be done this way, or can you just do everything in one dir using a multimodule reactor?